pkg/tagger/global.go

```diff
@@ -47,7 +47,9 @@ var tlmUDPOriginDetectionError = telemetry.NewCounter("dogstatsd", "udp_origin_d
 	nil, "Dogstatsd UDP origin detection error count")
 
 // Init must be called once config is available, call it in your cmd
-func Init() {
+func Init() error {
+	var initErr error
```

Comment:
Should this be a global?  Otherwise calls to Init after the first (of which there must be some, or `sync.Once` wouldn't be necessary) will return no error when in fact an error occurred.